views/events.ejs

```diff
@@ -217,23 +181,7 @@
       openModal();
     }
   });
-  function openEditEventModal(eventData) {
-    // Parse the JSON string to get the event object
-    const event = JSON.parse(eventData);
-
-    // Use the event data to populate the edit form
-    document.getElementById('eventname-e').value = event.eventname;
-    document.getElementById('eventdescription-e').value = event.eventdescription;
-    document.getElementById('eventdate-e').value = event.eventdate.split('T')[0]; // Extract date part
-    document.getElementById('eventtime-e').value = event.eventtime;
-    document.getElementById('eventvenue-e').value = event.eventvenue;
-    document.getElementById('eventorganizer-e').value = event.eventorganizer;
-    document.getElementById('eventorganizeremail-e').value = event.eventorganizeremail;
-    document.getElementById('eventorganizerphonenumber-e').value = event.eventorganizerphonenumber;
-
-    // Show the edit event modal
-    document.getElementById('editEventModal').style.display = 'block';
-  }
+
   // function to display create event
   function openAddEventModal() {
```

Comment:
Instead of `function` keyword try to use arrow functions as much unless it's needed?